stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
I think `env()` would make more sense.

On the other hand, LLVM does identify Android as an "environment" rather than an "OS" but I can understand that from a compiler/ABI perspective.

At least for Cygwin, using `env()` rather than `os()` is a lot more sensible. Otherwise you'd see `#if os(Windows) || os(Cygwin)` appearing all over the place unnecessarily.